### PR TITLE
[enterprise-4.7] Updating BZ-1903943-4.7

### PR DESCRIPTION
**Pull request by kdrake20**
Fixes: [BZ-1903943](https://bugzilla.redhat.com/show_bug.cgi?id=1903943)

From https://github.com/openshift/openshift-docs/pull/36682

For: Version 4.7

Doc Preview: [deploy-preview-44881--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes](https://deploy-preview-44881--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes)

Signed-off-by: Katie Tothill ktothill@redhat.com

### CODE REVIEW

**Comment by mrniranjan** [comment]
Looks Good To me.

**Review by jlojosnegros — APPROVED** [review]
/lgtm

**Comment by ktothill** [comment]
Who:
Jose Luis Ojosnegros, SME
Niranjan Mallapadi Raghavender, QE

Date and Time: April 25, 2022  13:19
Summary: Jose and Niranjan provided `lgtm` statuses for both PRs.

Next Steps: Requesting merges from Peer Review Squad.

File: modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc [hunk 1]
```diff
@@ -52,13 +52,78 @@ spec:
...
 <2> Use this field to configure the topology manager policy. Valid values are `none` (default), `best-effort`, `restricted`, and `single-numa-node`. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policies[Topology Manager Policies].
 
-[id="cnf-partitioning-the-cpus_{context}"]
-== Partitioning the CPUs
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="cnf-cpu-infra-container_{context}"]
+= Restricting CPUs for infra and application containers
```
Comment (kalexand-rh) [inline modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc:60]: You can't put a second H1 in a module. You need to split this module into three parts: the original content, and the concept/procedure split from #37432

**Review by kalexand-rh — COMMENTED** [review]
Please incorporate any other changes from #37432 as well.

File: modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc (hunk 1 above)
Comment (ktothill) [reply modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc]: Updated as suggested.

File: modules/cnf-cpu-infra-container.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+:_content-type: PROCEDURE
+[id="cnf-restricting-cpu-infra-container_{context}"]
+:_context: cnf-master
```
Comment (bergerhoffer) [inline modules/cnf-cpu-infra-container.adoc:6]: Should be just `:context:`

File: modules/cnf-cpu-infra-container.adoc [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+= Restricting CPUs for infra and application containers
+
+Generic housekeeping and workload tasks use CPUs in a way that may impact latency-sensitive processes. By default, the container runtime uses all online CPUs to run all containers together, which can result in context switches and spikes in latency. Partitioning the CPUs prevents noisy processes from interfering with latency-sensitive processes by separating them from each other. The following table describes how processes run on a CPU after you have tuned the node using the Performance Addon Operator:
```
Comment (bergerhoffer) [inline modules/cnf-cpu-infra-container.adoc:9]: may -> might

File: modules/cnf-cpu-infra-container.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+====
+The reserved and isolated CPU pools must not overlap and together must span all available cores in the worker node.
+====
+
+To ensure that housekeeping tasks and workloads do not interfere with each other, specify two groups of CPUs in the `spec` section of the performance profile.
+
+* `isolated` - Specifies the CPUs for the application container workloads. Workloads running on these CPUs experience the lowest latency as well as zero interruptions and can, for example, reach high zero packet loss bandwidth.
+* `reserved` - Specifies the CPUs for the cluster and operating system housekeeping duties. Threads in the `reserved` group are often busy. Do not run latency-sensitive applications in the `reserved` group. Latency-sensitive applications run in the `isolated` group.
+
+.Procedure
```
Comment (bergerhoffer) [inline modules/cnf-cpu-infra-container.adoc:52]: Not critical, but there was quite a lot of lead-in information in this module that I was surprised it went into a procedure here. Might want to consider breaking out the majority of the above info into a separate concept module.

**Review by bergerhoffer — COMMENTED** [review]
Just a few more things to consider

File: modules/cnf-cpu-infra-container.adoc (hunk 2 above)
Comment (ktothill) [reply modules/cnf-cpu-infra-container.adoc]: Updated as suggested.

File: modules/cnf-cpu-infra-container.adoc (hunk 3 above)
Comment (ktothill) [reply modules/cnf-cpu-infra-container.adoc]: Updated as suggested.

File: modules/cnf-cpu-infra-container.adoc (hunk 4 above)
Comment (ktothill) [reply modules/cnf-cpu-infra-container.adoc:16]: Updated as suggested.

**Comment by tradej** [comment]
@ktothill this has been without movement for 16 days, what's the hold-up?

**Pull request merged**